app/scripts/controllers/network/network-controller.test.js

```diff
@@ -618,7 +601,7 @@ describe('NetworkController', () => {
           );
         });
 
-        it('emits infuraIsUnblocked (assuming that the request to eth_blockNumber responds successfully)', async () => {
+        it('emits infuraIsBlocked or infuraIsUnblocked, depending on whether Infura is blocking requests', async () => {
```

Comment:
should this block also test a case where Infura is blocked as the new name suggests?

Reply:
Well, my goal here was to test that `lookupNetwork` gets called. I think I do have a test for `lookupNetwork` specifically that confirms that `infuraIsBlocked` is called when Infura is blocking and that `infuraIsUnblocked` is called when it's not. But if it makes more sense to test `infuraIsBlocked` instead of `infuraIsUnblocked` I can do that here.